az-cvm-vtpm/src/vtpm/mod.rs

```diff
@@ -109,6 +111,9 @@ pub fn get_report_with_report_data(report_data: &[u8]) -> Result<Vec<u8>, Report
     let nv_index_report_data = NvIndexTpmHandle::new(INDEX_REPORT_DATA)?;
     write_nv_index(&mut context, nv_index_report_data, report_data)?;
 
+    // Sleep time 3s for the user data to be reflected in VTPM_HCL_REPORT_NV_INDEX
```

Comment:
Can we add a remark about the 3s to the fn doc?

Reply:
Sure - which file ?